src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Sample.java

```diff
@@ -70,4 +75,11 @@ public Long getId() {
     public AccessionVersionId getAccessionVersionId() {
         return accessionVersionId;
     }
+
+    @PrePersist
```

Comment:
if we are using prepersist we need to include preupdate and preremove as well

Reply:
Done